car/car.ino

```diff
@@ -129,3 +122,71 @@ void handleInput() { //handle serial input (String!!)
     }  
   }
 }
+
+void driveAroundObstical(){
+  int leftDistance = left.readRangeContinuousMillimeters();
+  int rightDistance = right.getMedianDistance();
+  
+  if (leftDistance > LEFT_DIST && rightDistance > RIGHT_DIST){               //if there is no obstical on either sides rotate 90 degrees to the right
```

Comment:
L130 does not do what the comment says it does. If  there are no obstacles you may very well get `0` as the `rightDistance`